core/src/actions/schedules.ts

```diff
@@ -1,5 +1,6 @@
 import { Schedule } from "../models/Schedule";
 import { AuthenticatedAction } from "../classes/actions/authenticatedAction";
+import { ConfigWriter } from "../modules/configWriter";
```

Comment:
This is the pattern you're going to see throughout the app. After first putting the responsibility on the model to trigger the ConfigWriter, I moved it to the actions. It feels more like a controller method because the controller can more appropriately accept parameters from the client to specify whether or not we should do a write.

Reply:
That's a great idea. I think I put this together before I knew about the mode variable. I like the pattern you're suggesting better. I'll put that in place.